lib/beaker/options/subcommand_options_file_parser.rb

```diff
@@ -3,18 +3,30 @@ module Options
     #A set of functions to read options files
     module SubcommandOptionsParser
 
+      HOMEDIR_OPTIONS_FILE_PATH = ENV['HOME']+'/.beaker/subcommand_options.yaml'
+
+      def self.parse_options_file(options_file_path)
+        result = OptionsHash.new
+        if File.exist?(options_file_path)
+          result = YAML.load_file(options_file_path)
+        end
+        result
+      end
+
       # @return [OptionsHash, Hash] returns an empty OptionHash or loads subcommand options yaml
       #   from disk
-      def self.parse_subcommand_options(argv)
+      def self.parse_subcommand_options(argv, home_dir=false)
```

Comment:
@kevpl updated code to use `file_option` argument and using constants for both, homedir file and subcommands options file.